### PR TITLE
update docs folder links

### PR DESCRIPTION
**Pull request by Arhell**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  |

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@bizley

File: docs/guide-fr/installation.md [hunk 1]
```diff
@@ -3,11 +3,11 @@ Installation
 
 ## Requirements
 
-l'extention  [MongoDB PHP Extension](http://us1.php.net/manual/en/set.mongodb.php) version 1.0.0 ou plus.
+l'extention  [MongoDB PHP Extension](https://us1.php.net/manual/en/set.mongodb.php) version 1.0.0 ou plus.
```
Comment (bizley) [inline docs/guide-fr/installation.md:6]: Please change it to the current one - https://www.php.net/manual/en/set.mongodb.php

Comment (Arhell) [reply docs/guide-fr/installation.md]: done

**Review by bizley — APPROVED** [review]

**Comment by bizley** [comment]
👍🏻

**Pull request merged**